tac/project-updates/2025/2025-Q3-Lockness.md

```diff
@@ -0,0 +1,65 @@
+
+# Project Health
+
+The project is in a healthy and active state, with development focused on two major efforts: a significant upgrade to the CGGMP core protocol and a foundational rework of the `round-based` library. The CGGMP update has already resulted in new alpha releases, while the crucial `round-based` update is a key work-in-progress. Our community is engaged and growing; questions are being answered on Discord and GitHub, and we've seen a new, highly active outside contributor make significant contributions this quarter.
+
+# Questions/Issues for the TAC
+
+In Discord, when people reach out to ask questions in the public channel, sometimes a scam bot tries impersonate stuff by posting a message like "please use this link to open a support ticket", and after a few minutes deletes a message. 
+
+I'd like to bring your attention to this issue. Assuming other channels fall victim for this, it looks like a massive attack. It'd be great if we could deploy any countermeasures.
+
+# Releases
+
+* cggmp24 [github](https://github.com/LFDT-Lockness/cggmp21) [crates-io](https://crates.io/crates/cggmp24/0.7.0-alpha.1)
+  * v0.7.0-alpha.1 (Published on: 2025-08-20)
+* cggmp21 [github](https://github.com/LFDT-Lockness/cggmp21) [crates-io](https://crates.io/crates/cggmp21)
+  * v0.6.2 (Published on: 2025-06-04)
+  * v0.6.1 (Published on: 2025-06-02)
+* cggmp24-keygen [github](https://github.com/LFDT-Lockness/cggmp21) [crates-io](https://crates.io/crates/cggmp24-keygen/0.7.0-alpha.1)
+  * v0.7.0-alpha.1 (Published on: 2025-08-20)
+* paillier-zk [github](https://github.com/LFDT-Lockness/cggmp21) [crates-io](https://crates.io/crates/paillier-zk)
+  * v0.7.0-alpha.1 (Published on: 2025-08-20)
+* key-share [github](https://github.com/LFDT-Lockness/cggmp21) [crates-io](https://crates.io/crates/key-share)
+  * v0.6.1 (Published on: 2025-06-02)
+* round-based [github](https://github.com/LFDT-Lockness/round-based) [crates-io](https://crates.io/crates/round-based)
+  * v0.4.1 (Published on: 2025-01-21)
+* fast-paillier [github](https://github.com/LFDT-Lockness/fast-paillier) [crates-io](https://crates.io/crates/fast-paillier)
+  * v0.1.1 (Published on: 2025-04-15)
+* generic-ec [github](https://github.com/LFDT-Lockness/generic-ec) [crates-io](https://crates.io/crates/generic-ec)
+  * v0.4.5 (Published on: 2025-01-20)
+* generic-ec-zkp [github](https://github.com/LFDT-Lockness/generic-ec) [crates-io](https://crates.io/crates/generic-ec-zkp)
+  * v0.4.4 (Published on: 2025-01-28)
+* generic-ec-curves [github](https://github.com/LFDT-Lockness/generic-ec) [crates-io](https://crates.io/crates/generic-ec-curves)
+  * v0.2.4 (Published on: 2025-01-20)
+* generic-ec-core [github](https://github.com/LFDT-Lockness/generic-ec) [crates-io](https://crates.io/crates/generic-ec-core)
+  * v0.2.3 (Published on: 2025-01-20)
+* hd-wallet [github](https://github.com/LFDT-Lockness/hd-wallet) [crates-io](https://crates.io/crates/hd-wallet)
+  * v0.6.1 (Published on: 2025-01-17)
```

Comment:
Do you mind just adding the released for this quarter only?

Reply:
^^ +1